### ISSUE TITLE
Option to avoid using IP instead of hostname

### ISSUE THREAD

**Issue by tomasAlabes** [buger/goreplay#922]
Hi, when trying to replay a set of requests from a file, the resolution of the hostname to the IP makes the replay impossible (as my web server doesn't let me). I couldn't find an option to disable the IP or always use the hostname.

```shell
./gor --input-raw :80 --http-allow-url /api --output-file requests.log

./gor --input-file "requests_0.log" --output-http http://my.site/ --verbose 1
2021/05/06 12:20:17 [PPID 20391 and PID 20392] Version:1.2.0
[DEBUG][elapsed 5.002749884s]: [HTTPClient] Connection error: dial tcp XXX.XXX.XXX.XXX:80: i/o timeout
```

What can I do to workaround this?
Thanks!

**Comment by buger** [comment]
Hi! 

Frankly I'm not quite sure what you mean. 
While replaying GoReplay acts as a standard HTTP client. And replace "original" host with the host specified in output http argument. 
If you do curl call http://my.site/ URL will it work? 

If there is one tuning available in this logic, to allow keep "original" HTTP Host header, which can be enabled using `--http-original-host` flag. Hope it helps!

**Comment by tomasAlabes** [comment]
Hi, I ended up using another tool but thanks for answering.
I recorded my requests going to my.site and wanted to replay them also to my.site (I used the basic starter in the docs). But the logs showed the errors with the IP instead of the domain name and didn't work.
I think I tried with `http-original-host` but no luck.

Feel free to close the issue if you feel this was a mistake from my side (probably was as it is a basic case).

Thanks
